e2e-tests/functions

```diff
@@ -312,7 +317,27 @@ wait_cluster_consistency() {
 	"$(kubectl get ps "${cluster_name}" -n "${NAMESPACE}" -o jsonpath='{.status.orchestrator.ready}')" == "${orc_size}" &&
 	"$(kubectl get ps "${cluster_name}" -n "${NAMESPACE}" -o jsonpath='{.status.orchestrator.state}')" == "ready" &&
 	"$(kubectl get ps "${cluster_name}" -n "${NAMESPACE}" -o jsonpath='{.status.state}')" == "ready" ]]; do
-		echo 'waiting for cluster readyness'
+		echo 'waiting for cluster readyness (async)'
```

Comment:
If we use this method for async, should we rename it?

Reply:
Makes sense! I have renamed it and removed in some places, let's see how the tests go.